stan/math/prim/meta/is_container.hpp

```diff
@@ -19,7 +20,8 @@ namespace stan {
  */
 template <typename Container>
 using is_container = bool_constant<
-    math::disjunction<is_eigen<Container>, is_std_vector<Container>>::value>;
+  math::disjunction<is_eigen<Container>, is_std_vector<Container>,
+		    is_var_matrix<Container>>::value>;
```

Comment:
A `var<mat>` should count as a container, right? This might break template logic elsewhere but we can fix it elsewhere.